test/test_return.py

```diff
@@ -161,6 +161,14 @@ def test_return_location_edit(self):
         ), 'location method not allowed'
         assert self.get()['headers']['Location'] == 'blah'
 
+        assert 'success' in self.conf(
+            {"return": 302, "location": "foo${host}baz"}, 'routes/0/action'
```

Comment:
For me an Integration Test always helps to understand how the software engineers behind the test wants the software to be used. In this case we are using variable values like `foo` `baz` and `blahh` which technically works totally fine. BUT just think about the test with this code
```python
 {"return": 302, "location": "http://${host}"}, 'routes/0/action'
```
and the assert
```python
 assert (
            self.get(headers={"Host": "foobaz.local", "Connection": "close"}
            )['headers']['Location'] == 'http://foobaz.local'
        ), 'location with a variable'
```
This would showcase and test a real world implementation using an protocol handle in the URI and shows the users how to use the location correctly.

From my point of view the integration test needs to be as close on a real world use case as possible.

Reply:
Thanks, will try to make it more meaningful.